src/ir/term/test.rs

```diff
@@ -14,6 +15,37 @@ fn eq() {
     assert!(u != w);
 }
 
+#[test]
+fn map_eq_test() {
+    let a1 = 
+        make_array(Sort::BitVector(32), Sort::Bool,
+            vec![bool(true), bool(true), bool(false), bool(false)]);
+    let a2 = 
+        make_array(Sort::BitVector(32), Sort::Bool,
+            vec![bool(true), bool(false), bool(true), bool(false)]);
+    let t = term![Op::Map(Box::new(Op::Eq)); a1, a2];
+
+    let val = eval(&t, &FxHashMap::default());
+    println!("Value is: {}", val);
```

Comment:
change to `assert!` on an array term with value: `vec![bool(true), bool(false), bool(false), bool(false)]`